tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -788,26 +788,80 @@ GDScript uses different variable types than GLSL does, so when passing variables
 from GDScript to shaders, Godot converts the type automatically. Below is a
 table of the corresponding types:
 
-+-----------------+-----------+
-| GDScript type   | GLSL type |
-+=================+===========+
-| **bool**        | **bool**  |
-+-----------------+-----------+
-| **int**         | **int**   |
-+-----------------+-----------+
-| **float**       | **float** |
-+-----------------+-----------+
-| **Vector2**     | **vec2**  |
-+-----------------+-----------+
-| **Vector3**     | **vec3**  |
-+-----------------+-----------+
-| **Color**       | **vec4**  |
-+-----------------+-----------+
-| **Transform3D** | **mat4**  |
-+-----------------+-----------+
-| **Transform2D** | **mat4**  |
-+-----------------+-----------+
-
++----------------------+-------------------------+------------------------------------------------------------+
+| GLSL type            | GDScript type           | Notes                                                      |
++======================+=========================+============================================================+
+| **bool**             | **bool**                |                                                            |
++----------------------+-------------------------+------------------------------------------------------------+
+| **bvec2**            | **int**                 | Bitwise packed int where bit 0 (LSB) corresponds to x.     |
++----------------------+-------------------------+------------------------------------------------------------+
+| **bvec3**            | **int**                 | Bitwise packed int where bit 0 (LSB) corresponds to x.     |
++----------------------+-------------------------+------------------------------------------------------------+
+| **bvec4**            | **int**                 | Bitwise packed int where bit 0 (LSB) corresponds to x.     |
```

Comment:
Consider adding an example with GDScript here
i.e. 
For example to set ``bvec2(false, true)`` in the shader using the two bools ``bx = false`` and ``by = true`` in GDScript you would use ``(int(bx) | (int(by) << 1))``